tests/meta/clean/conftest.py

```diff
@@ -234,3 +236,127 @@ def hk_sample_bundle_for_flow_cell_not_to_clean(
         "expires": timestamp_yesterday,
         "files": [],
     }
+
+
+@pytest.fixture
+def clean_retrieved_spring_files_api(
+    real_housekeeper_api: HousekeeperAPI,
+) -> CleanRetrievedSpringFilesAPI:
+    """Returns a CleanRetrievedSpringFilesAPI"""
+    return CleanRetrievedSpringFilesAPI(real_housekeeper_api)
+
+
+@pytest.fixture
+def path_to_old_retrieved_spring_file() -> str:
+    return Path("path", "to", "old", "retrieved", "spring", "file").as_posix()
+
+
+@pytest.fixture
+def path_to_newly_retrieved_spring_file() -> str:
+    return Path("path", "to", "newly", "retrieved", "spring", "file").as_posix()
+
+
+@pytest.fixture
+def path_to_archived_but_not_retrieved_spring_file() -> str:
+    return Path("path", "to", "archived", "spring", "file").as_posix()
+
+
+@pytest.fixture
+def path_to_fastq_file() -> str:
```

Comment:
type hints are wrong, they say str but all fixtures return Path

Reply:
The `as_posix` converts Path to str

Reply:
right! my mistake